### PR TITLE
Hw 5.2 solution

### PR DESCRIPTION
**Pull request by SviatoslavExpert**

### CODE REVIEW

File: src/main/java/core/basesyntax/Car.java [hunk 1]
```diff
@@ -1,5 +1,113 @@
...
+        this.maxSpeed = maxSpeed;
+        this.modelName = modelName;
+        this.color = color;
+        this.manufacturer = manufacturer;
+        this.engine = engine;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+        super.clone();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Car.java]: do we need this line here?

File: src/main/java/core/basesyntax/Engine.java [hunk 2]
```diff
@@ -1,5 +1,75 @@
...
+        this.number = number;
+        this.horsePower = horsePower;
+        this.volume = volume;
+        this.manufacturer = manufacturer;
+        this.type = type;
+    }
+
+    @Override
+    public Engine clone() throws CloneNotSupportedException {
+        super.clone();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Engine.java]: the same

Comment (SviatoslavExpert) [reply src/main/java/core/basesyntax/Engine.java]: super.clone(); is removed.

File: src/main/java/core/basesyntax/Car.java (hunk 1 above)
Comment (SviatoslavExpert) [reply src/main/java/core/basesyntax/Car.java]: super.clone(); is removed.

**Pull request closed without merging**